CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ a release.
 
 ### Changed
 - Sluggable: Replaced abandoned `behat/transliterator` with `symfony/string` for default transliteration and urlization steps (#2985)
+- Use `getFieldValue` and `setFieldValue` methods to support `doctrine/orm` >= 3.4 (#2966)
```

Comment:
```suggestion
- Use `ClassMetadata::getFieldValue()` and `ClassMetadata::setFieldValue()` methods to support `doctrine/orm` >= 3.4 (#2966)
```

BTW, if the changes introduced in this PR are described as a fix for an existing issue (#2965), the `Fixed` section of the changelog MUST also be included, explaining what is fixed here.